pkgs/development/python-modules/outlines/default.nix

```diff
@@ -4,56 +4,60 @@
   fetchFromGitHub,
   setuptools,
   setuptools-scm,
+  airportsdata,
   interegular,
   cloudpickle,
   datasets,
   diskcache,
-  joblib,
+  jinja2,
   jsonschema,
-  pyairports,
+  numpy,
+  outlines-core,
   pycountry,
   pydantic,
   lark,
   nest-asyncio,
-  numba,
-  scipy,
+  referencing,
+  requests,
   torch,
   transformers,
 }:
 
 buildPythonPackage rec {
   pname = "outlines";
-  version = "0.0.46";
+  version = "0.1.4";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "outlines-dev";
     repo = pname;
     rev = "refs/tags/${version}";
```

Comment:
```suggestion
    repo = "outlines";
    tag = version;
```

Reply:
Fixed. Seems like support for passing through `tag` was from after this PR, but it worked after a rebase.